docs/content/latest/reference/configuration/yugabyted/errors-glossary.md

```diff
@@ -1,14 +1,16 @@
 ---
-title: yugabyted Errors Glossary
-linkTitle: Errors Glossary
-description: yugabyted CLI errors glossary
+title: yugabyted Errors and Warnings
+linkTitle: Errors and Warnings
+description: yugabyted CLI errors and warnings
 aliases:
-  - /latest/integrations/completeness-matrix
+  - /latest/integrations/yugabyted-errors-warnings
```

Comment:
Why? This is a new file, so just decide where it should live, and everything will link there.